releasenotes/notes/nova-13.0-0c2497128e677cb2.yaml

```diff
@@ -0,0 +1,20 @@
+---
+prelude: >
```

Comment:
That's the usual openstack way of writing release notes... Because text will be rendered "merged", the different sections will be splitted from the prelude. The prelude explain what has been happening in general, and the other explain the more non-obvious details.

In our case, we could ask ourselves if the prelude is relevant. I think it's important for our teams to know a new db was introduced, and to be clear and upfront about it.